### DISCUSSION TITLE
I can't build Kade Engine 1.5 or 1.6

### DISCUSSION THREAD

**Discussion by Fury1012** [KadeArchive/Kade-Engine#2771] [Q&A] [answered]
it always builds but when i try to run or test it, it always crashes when the cache loading stops
![image](https://user-images.githubusercontent.com/45504212/146649593-f2c49150-675b-4039-b55e-6d9af7599577.png)

**Comment by Fadeyauso** [comment, marked as answer]
same

**Reply by glowsoony** [reply]
ok the problem here is on titlescreen.hx if you see the  lines saying:
#if polymod
polymod.Polymod.init({modRoot: "mods", dirs: ['introMod']});
#end 
delete all of the 3 lines and you should test it now and it should open :)
the reason to delete these is its useless and is what makes your game crash.

**Reply by Fury1012** [reply]
oh my god, thank you so much.

**Reply by AbnormalPoof** [reply]
Or just install polymod via haxelib
